source/libraries/settings/board.js

```diff
@@ -6,7 +6,7 @@ var board = {
 	{
 		linux:
 		{
-			serial:'/dev/ttyAMA0_offline',
+			serial:'/dev/ttyS0',
```

Comment:
I think this port is different from for some Raspberry Pi version, I would add this as a command line parameter rather than taking it from here. 

We disabled this on purpose, but I cannot exactly remember why.

Reply:
Yes, you are right. For default, in Raspberry Pi Zero, 1 and 2 is ttyAMA0. And, in Raspberry Pi Zero W, 3 and 4 is ttyS0. [UART Raspberry Pi](https://www.raspberrypi.org/documentation/configuration/uart.md).

I think that it is possible to use '/dev/ttyS0' as default, and change libraries/settings/index.js to add support to the command line option for this parameter. Something like that:

```js
var boardserial = process.argv[3];
if (boardserial)
{
    board[boardtype].serial = boardserial;
}
```

Reply:
please add this change too and let's merge it